src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -195,10 +195,10 @@ void CompilerOpenFPGA::Help(std::ostream* out) {
          << std::endl;
   (*out) << "   synth_options <option list>: Yosys Options" << std::endl;
   (*out) << "   pnr_options <option list>  : VPR Options" << std::endl;
-  (*out)
-      << "   pnr_netlist_lang <blif, edif, verilog, vhdl> : Chooses vpr input "
-         "netlist format"
-      << std::endl;
+  (*out) << "   pnr_netlist_lang <blif, eblif, edif, verilog, vhdl> : Chooses "
+            "vpr input "
```

Comment:
There appears to be white spaces here and below.

Reply:
Updated.